### PR TITLE
RHCLOUD-24394 Displaying tags on the cards

### PR DESCRIPTION
**Pull request by josejulio**
screenshot_2023-03-14T12:05:18

### CODE REVIEW

File: src/components/Tags/Tag.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+const colorForLabelType = (type: APILabel['type']): LabelProps['color'] => {
+    switch (type) {
+        case "platform":
+            return 'green';
+        case "service":
+        case "use-case":
+            return 'blue';
+        default:
```
Comment (tahmidefaz) [inline src/components/Tags/Tag.tsx:17]: @josejulio  Question: Is the service case missing a return statement? Or, it it getting inferred from "use-case"?

**Review by tahmidefaz — CHANGES_REQUESTED** [review]
I just had a question. I am not sure if this was intended or if it's a syntax error.

File: src/components/Tags/Tag.tsx (hunk 1 above)
Comment (josejulio) [reply src/components/Tags/Tag.tsx:17]: It's using the same as service. I saw in the mocks that both were blue

**Review by tahmidefaz — APPROVED** [review]
Looks good!

**Pull request merged**